queries.sql

```diff
@@ -0,0 +1,59 @@
+-- Transaction 1: Set species to 'unspecified' for all animals, then rollback and verify
+START TRANSACTION;
+
+UPDATE animals SET species = 'unspecified';
+
+-- Verify the change (should show 'unspecified')
+SELECT * FROM animals;
+
+ROLLBACK;
+
+-- Verify the change is rolled back (should show NULL for species)
+SELECT * FROM animals;
+
+-- Transaction 2: Update species for specific animals, commit, and verify
+START TRANSACTION;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+-- Verify the changes
+SELECT * FROM animals;
+
+COMMIT;
+
+-- Verify changes persist after commit
+SELECT * FROM animals;
+
+-- Transaction 3: Delete all records, then rollback and verify
+START TRANSACTION;
+
+DELETE FROM animals;
+
```

Comment:
- [ ] Good job so far 👏 👏 however, in this project we were required to write queries to answer the following questions:
    - How many animals are there?
    - How many animals have never tried to escape?
    - What is the average weight of animals?
    - Who escapes the most, neutered or not neutered animals?
    - What is the minimum and maximum weight of each type of animal?
    - What is the average number of escape attempts per animal type of those born between 1990 and 2000?


and also these queries should be added to your `queries.sql` file and to add screenshots with the results of your queries 👍 to complete the project requirements